sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Azure.Monitor.OpenTelemetry.LiveMetrics.csproj

```diff
@@ -13,7 +13,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
     <PackageReference Include="OpenTelemetry" />
-    <PackageReference Include="OpenTelemetry.Exporter.Console" VersionOverride="1.6.0" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" />
```

Comment:
Can't we achieve this feature without this reference?

Reply:
It could, but I don't think we want to.

This PR delivers a single extension method that configures LiveMetrics for both signals.
This extension method uses `OpenTelemetryBuilder` which comes from `OpenTelemetry.Extensions.Hosting`.
https://github.com/Azure/azure-sdk-for-net/blob/a11ec53bded91ed65689c803c34ab62c83885e73/sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/LiveMetricsExtensions.cs#L30

Without this package, a user would need to configure our `Manager` singleton to the ServiceCollection and set LiveMetrics on both signals. I don't think we should take this approach.

Second, as we're preparing to shift our implementation to the Distro, it makes sense to take common dependencies now. This will make that shift easier.